master/buildbot/worker_transition.py

```diff
@@ -199,7 +201,9 @@ def deprecatedWorkerModuleAttribute(scope, attribute, compat_name=None,
         "scope must be module, i.e. locals()"
 
     if new_name is None:
-        attribute_name = scope.keys()[scope.values().index(attribute)]
+        scope_keys = list(iterkeys(scope))
```

Comment:
if we need a list, there is no need to use the future util.

We can just do list(scope.keys()) which will work well on py2 and py3

even if it might be less optimal, I think it is more readable